<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventDetails</class>
 <widget class="QWidget" name="EventDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QLabel" name="l_Equipement">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Equipement</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="l_Type">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" rowspan="2" colspan="2">
    <widget class="QDateTimeEdit" name="dt_Event">
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QLineEdit" name="le_Titre"/>
   </item>
   <item row="2" column="2" rowspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Titre:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_9">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Commentaires:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QTextEdit" name="te_Comments"/>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QStackedWidget" name="sw_Sip_Pro">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_Vide"/>
     <widget class="QWidget" name="page_Profil">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <widget class="QSpinBox" name="sp_NumProfil">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>N° profil</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Référence célérité:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Fichier :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_FicProfil"/>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cb_RefCelerite">
         <item>
          <property name="text">
           <string>Thermo-Salinomètre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Célérimètre</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_Sippican">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="le_probeType"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Type de sonde</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Tir Sippican</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="le_ProbeNumber"/>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="cb_ChargeSIS">
         <property name="text">
          <string>Chargé dans SIS à:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="le_FicASVP"/>
       </item>
       <item row="8" column="1">
        <widget class="QDateTimeEdit" name="dt_ChargeSIS"/>
       </item>
       <item row="6" column="4">
        <widget class="QDoubleSpinBox" name="sp_CelASVP">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>1400.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1600.000000000000000</double>
         </property>
         <property name="value">
          <double>1500.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Célérité ASVP à 3m (m/s):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Fichier ASVP:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Fichier SIS:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QLineEdit" name="le_FicSis"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Navire</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="sp_SOG">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Vitesse fond (nd)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="sp_COG">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Route fond (°)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="le_Position">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>47° 47.442' N, 3° 57.451' W</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Sonde (m)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="sp_Sonde">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Thermo-salinomètre</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Température (°C:)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="sp_SBETemp">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Salinité (psu):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="sp_SBESalinite">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="value">
         <double>35.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Célérité (m/s):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="sp_SBECelerite">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>1400.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1600.000000000000000</double>
        </property>
        <property name="value">
         <double>1500.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Célérimètre</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Célérité (m/s):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="sp_Celerimetre">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>1400.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1600.000000000000000</double>
        </property>
        <property name="value">
         <double>1500.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QStackedWidget" name="sw_btns">
     <property name="maximumSize">
      <size>
       <width>214</width>
       <height>42</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>164</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="btn_UnLock">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icones/CadenasFerme.jpg</normaloff>:/icones/CadenasFerme.jpg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_Modifier">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_Supprimer">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="btn_Lock">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icones/CadenasOuvert.jpg</normaloff>:/icones/CadenasOuvert.jpg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_Valider">
         <property name="maximumSize">
          <size>
           <width>95</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_Annuler">
         <property name="maximumSize">
          <size>
           <width>95</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
